virtool/samples/fake.py

```diff
@@ -102,6 +103,14 @@ async def create_fake_sample(
                     sample_id,
                 )
 
+    settings = Settings()
+    settings.app = app
+    settings.db = db
```

Comment:
Why is this being done?

Reply:
I just updates the code that was already there, before the "app" and "db" values were assigned to setting s like this:
`    sample = await create_sample(
        _id=sample_id,
        db=db,
        name=f"Fake {sample_id.upper()}",
        host="Vine",
        isolate="Isolate A1",
        locale="",
        subtractions=subtraction_ids,
        notes=fake.text(50),
        library_type="normal",
        labels=[],
        user_id=user_id,
        group="none",
        settings={
            "app": app,
            "db": db,
            "sample_group_read": True,
            "sample_group_write": True,
            "sample_all_read": True,
            "sample_all_write": True,
        },
    )
`

Reply:
OK. Doesn't make sense to me. Can you check if it is necessary and remove if it isn't?

Reply:
I just checked and the "app" and "db" values aren't used so I removed those two lines